src/Client/Suggest.cpp

```diff
@@ -82,6 +60,7 @@ static String getLoadSuggestionQuery(Int32 suggestion_limit, bool basic_suggesti
     add_column("name", "data_type_families", false, {});
     add_column("name", "merge_tree_settings", false, {});
     add_column("name", "settings", false, {});
+    add_column("keyword", "keywords", false, {});
```

Comment:
This breaks completion for older servers, can we add this to the query only if the reversion/version is new enough?

Reply:
Sure.